pkgs/development/python-modules/paramiko/default.nix

```
@@ -18,12 +18,12 @@
 
 buildPythonPackage rec {
```

Could you remove the `rec` keyword? (ref: https://nix.dev/guides/best-practices#recursive-attribute-set-rec)
You can define following to remove it.

```suggestion
let
  pname = "paramiko";
  version = "3.5.0";
in
buildPythonPackage {
```